src/modules/app/components/side-nav/side-nav.styles.less

```diff
@@ -84,9 +87,48 @@
     }
   }
 
+  .SideNav__nav__separator {
+    background: @color-lightpurple;
+    height: 0.0625rem;
+    margin: 1rem 0 0.5rem;
+    width: 12.125rem;
+  }
+
+  .SideNav__stat-label {
+    color: rgba(255, 255, 255, 0.25);
```

Comment:
Use less’ `fadeout` function w/ a var from our `colors.less` stylesheet.  

Totally OK to set an opacity percentage manually here.